.github/ISSUE_TEMPLATE/rfc_template.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "Request For Comments (RFC)"
+about: "You have an idea on how to improve our processes. Propose your idea so that the team can provide feedback."
+title: "RFC: "
+labels: "type : rfc"
+---
+
+## Issue
+
+Describe the issue the team is currently facing. Provide as much content as possible.
+
+## Solution
+
+Describe the solution you are prescribing for the issue
```

Comment:
Missing ending dot for this sentence, right? 
```suggestion
Describe the solution you are prescribing for the issue.
```

Reply:
lol @minhnimble it's ok the sentence will be removed entirely when you file a real RFC